icu4c/source/test/intltest/restsnew.cpp

```diff
@@ -197,6 +202,12 @@ void NewResourceBundleTest::runIndexedTest( int32_t index, UBool exec, const cha
     case 5: name = "TestGetByFallback";  if(exec) TestGetByFallback(); break;
     case 6: name = "TestFilter";  if(exec) TestFilter(); break;
 
+#if U_ENABLE_TRACING
+    case 7: name = "TestTrace";  if(exec) TestTrace(); break;
+#else
+    case 7: name = "skip"; break;
```

Comment:
Might be a good time to upgrade this file to the TEST_CASE_AUTO mechanism; then you don't need an empty skip invocation.